util/syspolicy/handler.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package syspolicy
+
+import (
+	"errors"
+	"sync/atomic"
+)
+
+var handler atomic.Value
```

Comment:
no need for an atomic?

Reply:
Apple platforms are going to work by having something in Swift implement the pieces needed for the Handler interface. I'm still a little fuzzy on the exact timing of when `RegisterHandler` (which seems to have been omitted from the PR at the moment, it'll need to come back) will be called. Once that's figured out, hopefully we can drop the atomic.

Reply:
At least use `syncs.AtomicValue[T]` so the type is obvious. (Prior to that, we used comments to say what the type of type-unsafe things were)